src/test/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepRestartTest.java

```diff
@@ -97,11 +99,7 @@ private void sanity(WorkflowRun b) throws Exception {
                 WorkflowRun b = j.jenkins.getItemByFullName("p", WorkflowJob.class).getBuildByNumber(1);
                 assertNotNull(b);
                 assertTrue(b.isBuilding());
-                Executor executor;
-                while ((executor = b.getExecutor()) == null) {
-                    Thread.sleep(100); // probably a race condition: AfterRestartTask could take a moment to be registered
```

Comment:
Dropping the explanatory comment, but I guess it is not controversial enough to bother.